frontend/src/scenes/insights/EditorFilters/EFFunnelsAdvanced.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import { useValues, useActions, useMountedLogic } from 'kea'
+import clsx from 'clsx'
+import { funnelLogic } from 'scenes/funnels/funnelLogic'
+import { Tag } from 'antd'
+import { InfoCircleOutlined } from '@ant-design/icons'
+import { Tooltip } from 'lib/components/Tooltip'
+import { FunnelStepReference, StepOrderValue, EditorFilterProps } from '~/types'
+import { IconArrowDropDown } from 'lib/components/icons'
+import { FunnelStepOrderPicker } from '../InsightTabs/FunnelTab/FunnelStepOrderPicker'
+import { FunnelExclusionsFilter } from '../InsightTabs/FunnelTab/FunnelExclusionsFilter'
+import { FunnelStepReferencePicker } from '../InsightTabs/FunnelTab/FunnelStepReferencePicker'
+import { funnelCommandLogic } from '../InsightTabs/FunnelTab/funnelCommandLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+
+export function EFFunnelsAdvanced({ filters, insightProps }: EditorFilterProps): JSX.Element {
+    const { aggregationTargetLabel, advancedOptionsUsedCount } = useValues(funnelLogic(insightProps))
+    const { setFilters, toggleAdvancedMode, setStepReference } = useActions(funnelLogic(insightProps))
+    useMountedLogic(funnelCommandLogic)
+
+    return (
+        <>
+            <hr />
+            <div className="flex-center cursor-pointer" onClick={toggleAdvancedMode}>
+                <h4 className="secondary" style={{ flexGrow: 1 }}>
+                    Advanced options{' '}
+                    {!filters.funnel_advanced && !!advancedOptionsUsedCount && (
+                        <Tag className="lemonade-tag">{advancedOptionsUsedCount}</Tag>
+                    )}
+                </h4>
+                <div>
+                    <div className={clsx('advanced-options-dropdown', filters.funnel_advanced && 'expanded')}>
+                        <IconArrowDropDown />
+                    </div>
+                </div>
+            </div>
+            {filters.funnel_advanced ? (
```

Comment:
Somehow this never opens.

Reply:
This is fixed now - removed all of that checking as it is basically handled by the parent component now